controller/comments.controller.js

```diff
@@ -0,0 +1,12 @@
+const modelDeleteComment = require("../model/comments.model");
+
+exports.deleteComment = (request, response, next) => {
+  const commentNum = request.params.comment_id;
+  modelDeleteComment(commentNum)
+    .then(() => {
+      response.status(204).send({ message: "Deletion Successful!" });
+    })
```

Comment:
No need for the message. No response is sent from a 204 so it can just be an empty object